docker/docker.go

```diff
@@ -25,6 +26,11 @@ const (
 )
 
 func main() {
+	// Set terminal emulation based on platform as required.
+	stdout, stderr, stdin := term.StdStreams()
+
+	initLogging(stderr)
```

Comment:
@icecrime 
I am not familiar with how reexec works. But my impression was that in case of the errors some of the initializers might write to the stderr. That was one reason why setting terminal and log output is done as soon as we can.